app/config.go

```diff
@@ -82,7 +82,7 @@ func (a *App) ClientConfigHash() string {
 	return a.Srv().ClientConfigHash()
 }
 
-func (a *App) LimitedClientConfig() map[string]string {
+func (a *App) limitedClientConfig() map[string]string {
 	return a.Srv().limitedClientConfig.Load().(map[string]string)
 }
```

Comment:
Wondering if it is necessary to even keep this methods? I think we can remove these and use `a.Srv()...` instead.

Reply:
You are totally right, this is not needed, I'll change that and remove any private method that only wraps the `Srv` call.

Reply:
Done. PTAL.